babel.config.js

```diff
@@ -1,6 +1,6 @@
-module.exports = function(api) {
-  api.cache(true);
+module.exports = (api) => {
+  api.cache(false);
```

Comment:
Do we really need to turn off caching?

Reply:
It caused the env variables not been updated when I was trying to swap the api url. This is because `react-native-dotenv` requires babel to load it as a module.